packages/data-stores/src/site/actions.ts

```diff
@@ -272,6 +272,15 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		yield saveSiteSettings( siteId, { blogdescription } );
 	}
 
+	function* setThemeOnSite( siteSlug: string, theme: string ) {
+		yield wpcomRequest( {
+			path: `/sites/${ siteSlug }/themes/mine`,
+			apiVersion: '1.1',
+			body: { theme: theme, dont_change_homepage: true },
```

Comment:
Note to self: What would happen if `dont_change_homepage` was set to `false`. We definitely want the "Hello World!" post to be kept.

![image](https://user-images.githubusercontent.com/1287077/178933704-94e387ce-582c-4f3d-bccb-9f189039c1a2.png)